index.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+var graylog2 = require('graylog2');
+var Squeeze = require('good-squeeze').Squeeze;
+
+function GraylogReporter(events, config) {
+  this.client = new graylog2.graylog({
+    servers: [{ host : config.host, port: config.port }],
+    facility: config.service,
+    hostname: (config.container) ? config.container : undefined
```

Comment:
If this is `hostname` we should probably have a more complete set of defaults that includes the `HOSTNAME` environment variable.